scalars.Rmd

```diff
@@ -56,4 +56,283 @@ $$
 \bar{a} {+=} \bar{c}
 \qquad
 \bar{b} {+=} -\bar{c}
```

Comment:
I think `\bar` may be OK with simple letters.  Looks like I did that in the initial part of this file.  But I had to use `\overline` elsewhere to get it to look right.